repository-data/application/src/main/resources/hcm-content/content/gallery/lks/hero-images.yaml

```diff
@@ -0,0 +1,34 @@
+/content/gallery/lks/hero-images:
```

Comment:
Could we move this under `repository-data/development` ?

Reply:
had trouble with this, keeps breaking my local repo